lib/atmega328/max72xx.cpp

```diff
@@ -6,18 +6,20 @@
 
 #define CS_PIN PB2
 
-void max72xx_write_byte(uint8_t reg, uint8_t data);
+void max72xx_write_byte(uint8_t device, uint8_t reg, uint8_t data);
```

Comment:
Use the type `max72xx_reg_t` so we get type checking

```suggestion
void max72xx_write_byte(uint8_t device, max72xx_reg_t reg, uint8_t data);
```

Reply:
I am removing the write_byte function so this case does not apply, but I will keep it in mind for other functions.